### DISCUSSION TITLE
proposal

### DISCUSSION THREAD

**Discussion by ghost** [tuskyapp/Tusky#3031] [Ideas]
proposal

![profil_pencil](https://user-images.githubusercontent.com/26837921/206656693-a7d97989-6c74-46b6-b2bc-3d5267a44b72.png)
![anasayfa](https://user-images.githubusercontent.com/26837921/206656699-4164bc11-f04d-44cd-8c11-d4cf5d24b2ba.png)
![emoji_search](https://user-images.githubusercontent.com/26837921/206656704-272d3ed2-4d85-42b1-8f6f-dd3b6399df6c.png)
].

* * * *
- Tusky Version:
- Android Version:
- Android Device:
- Mastodon instance (if applicable):

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by ghost** [comment]
![Search](https://user-images.githubusercontent.com/26837921/206664572-4864f944-5a61-4199-9ef2-f587534f51ad.png)

**Reply by charlag** [reply]
Hi
could you please verbalize your proposal a bit more? thank you

**Comment by ghost** [comment]
Merhaba, "homepage" (anasayfa), da olduğu gibi profile account" (profil hesabı) da, "sharing icon" (paylaşım simgesi),nin olması için bir özellik ekleyebilirsiniz. Bu özellik sayesinde kullanıcılar profil hesaplarındaki paylaşımları diğer kullanıcılarla paylaşabilirler.

Ayrıca, emoji search" (emoji arama),  özelliği de ekleyebilirsiniz. Bu özellik sayesinde kullanıcılar, emoji'leri kolayca arayarak kullanabilirler.

Logo yanında anasayfa yazı renkleri olarak "white" (beyaz) ve "purple" (mor) seçebilirsiniz. Bu renkler arasındaki kontrast sayesinde logo daha net görünecektir.

Logonun yanında tıklamadan otomatik arama özelliği de ekleyebilirsiniz. Bu özellik sayesinde kullanıcılar yazdıkları kelimeyi otomatik olarak arayabilirler.
{ En : Hello, in the "profile account", as in the "homepage", you can add a feature so that there is a "sharing icon". Thanks to this feature, users can share the shares in their profile accounts with other users.
In addition, you can also add the "emoji search", feature Dec. Thanks to this feature, users can easily use emojis by searching for them.
Besides the logo you can choose "white" and "purple" as the home page font colors. Thanks to the contrast between these colors, the logo will look more Decisively.
You can also add an auto-search feature without clicking next to the logo Dec. Thanks to this feature, users can automatically search for the word they have typed. }
